### DISCUSSION TITLE
【测试】今天注册成功，能看到朋友们帮忙扣个1谢谢

### DISCUSSION THREAD

**Discussion by Drunken-bassist** [share121/inter-knot#2507] [其他]
今天注册成功，能看到朋友们帮忙扣个1谢谢

**Comment by moxvan** [comment]
114514

**Reply by Drunken-bassist** [reply]
1919810

**Comment by 7346454** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >1

**Reply by Drunken-bassist** [reply]
：）

**Comment by 7346454** [comment]
1<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/94252e623d6ce6c3a2463be2afc23751_1000866240408823916.gif" alt="爱莉希雅-扭扭" width="75" >

**Reply by Drunken-bassist** [reply]
：）

**Comment by wz5204021** [comment]
给你点赞

**Reply by Drunken-bassist** [reply]
：）

**Reply by wz5204021** [reply]
1111111111111111

**Comment by zzz-Rope** [comment]
1<img src="https://bbs-static.miyoushe.com/static/2023/05/29/14066ebc42dde090117ffb7d23c214d8_6632143531559164244.png" alt="兰那罗-力量" width="75" >
![img_-1922540593_1723427273184](https://github.com/user-attachments/assets/ba4ab9df-6f2f-4e68-9348-eee8ff0ec44f)

**Comment by Bai6869** [comment]
1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111

**Comment by JobNumber-9527** [comment]
我是聋子，没看到所以我扣二，3<img src="https://bbs-static.miyoushe.com/static/2024/02/06/a06b39a87b3fec5be0f4ecbf4f758cad_5751963257293893910.png" alt="胡桃-收好啦" width="75" >

**Comment by 0wuming0** [comment]
1

**Comment by 00zzxx** [comment]
10086

**Comment by Kioeco** [comment]
114514

**Comment by Brighttaz** [comment]
b

**Comment by BHGD-031212** [comment]
1<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >
